implants/lib/eldritch/src/sys.rs

```diff
@@ -126,4 +127,8 @@ fn methods(builder: &mut MethodsBuilder) {
         if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
         get_reg_impl::get_reg(starlark_heap, reghiv, regpth)
     }
+    fn write_reg(this:  SysLibrary, reghiv: String, regpth: String, regname: String, regtype: String, regvalue: String) ->  anyhow::Result<String> {
```

Comment:
Let's drop the String return type for `()`
The eldritch way to know if you're updating or creating would be to proactively check if that key exists before writing.